README.md

```diff
@@ -31,12 +25,23 @@ which is the dependency of `chatting-client` and `chatting-server`.
 > But it is not mandatory for you to follow this architecture.
 > You can move the model codes to other places if you want.
 
-The first thing you need to do is to install the parent pom into the local maven repository.
+Now, the first thing you need to do is to install the parent pom into the local maven repository.
 
 ```shell
 mvn install
```

Comment:
please also help change `mvn` to `./mvnw` for linux/macOS, and `mvnw` for windows?